Unix/configure

```diff
@@ -1344,11 +1344,50 @@ rm -f $tmpdir/pthread_rwlock_t_func
 
 ##==============================================================================
 ##
+## Check whether SSL 1.0.x is installed on AIX platforms
+##
+##==============================================================================
+
+echo $echon "checking if SSL 1.0.x is installed on AIX platform ... $echoc"
```

Comment:
shouldn't the echo be inside the if statement so it is not echoed on other platforms?